app.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const {getCategories} = require('./controllers/games-controllers');
+
+const app = express();
+
+app.get('/api/categories', getCategories);
+
```

Comment:
It doesn't look like you've added error handling for invalid routes here, but you are testing for it.